src/pages/api/events/eventsFromEventServer.js

```diff
@@ -6,13 +6,21 @@ export default async function handler(req, res) {
 
   switch (method) {
     case "GET":
-      try {
-        const eventsFromEventServer = await prisma.event.findMany({});
+      // try {
```

Comment:
uncomment this part along with the catch block

Reply:
uncommented